tests/lang_tests_common.rs

```diff
@@ -16,15 +16,21 @@ fn main() {
     env::set_var("LD_LIBRARY_PATH", gcc_path);
     LangTester::new()
         .test_dir("tests/run")
-        .test_file_filter(|path| path.extension().expect("extension").to_str().expect("to_str") == "rs")
+        .test_file_filter(|path| {
```

Comment:
I'd appreciate if you remove unrelevant changes (formatting in this case).

I tried creating a config for rustfmt that would look like what I wanted, but I was unable to do so, so I gave up, but if that would be important to you and others, I could create issues in rustfmt's repo to request adding the options I'm missing.

Reply:
Have you considered adding a `.rustfmt.toml` file with `ignore = ...` or `disable_all_formatting = true`?

A number of people (including me) have auto-format-on-save enabled, and there's at least [one editor plugin](https://github.com/mitranim/sublime-rust-fmt/blob/86b3498adee26a278908df00f307e601e253dbf0/RustFmt.sublime-settings#L32) that enables this by default. Adding this would save churns for everyone and for you.

Reply:
Good idea, I did it [here](https://github.com/rust-lang/rustc_codegen_gcc/commit/dc8da94d56435f95547ac02cf634dea223668a78).